Documentation/DataTypes/Properties/GetText.rst

```diff
@@ -675,7 +675,7 @@ siteLanguage
    :ts:`direction`        The direction for this language
    :ts:`flagIdentifier`   The flag key (like "gb" or "fr") used to be used in TYPO3's Backend.
    :ts:`hreflang`         Language tag for this language defined by RFC 1766 / 3066 for "lang"  and "hreflang" attributes
-   :ts:`languageId`       The language mapped to the sys_language DB entry.
+   :ts:`languageId`       The language mapped to the site languge.
```

Comment:
Did you test the output of this? I'm not quite sure what a languageID would be in the context of site languages.

Reply:
I did test it and add where the id is coming from